Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -320,14 +319,19 @@ class SourceGenerator {
     }
 
     /// Checks whether the path is not in any default or TargetSource excludes
-    func isIncludedPath(_ path: Path) -> Bool {
+    func isIncludedPath(_ path: Path, targetSourceExcludePaths: Set<Path>, targetSourceIncludePaths: Set<Path>) -> Bool {
```

Comment:
For readability could we rename all the uses of these params to just excludePaths and includePaths